Какой будет результат выполнения следующего кода? Почему?
b = 10

def f(a):
    print(a)
    print(b)
    b = 15

f(3)

ANSWER: Код вызовет ошибку UnboundLocalError,
потому что оператор печати для b происходит до того,
как b присваивается внутри функции.
Python рассматривает b как локальную переменную из-за присваивания внутри функции,
но на момент оператора печати ей не было присвоено значение.
